tv.plex.PlexHTPC.yml

```diff
@@ -0,0 +1,106 @@
+app-id: tv.plex.PlexHTPC
+runtime: org.freedesktop.Platform.GL.default
```

Comment:
this is not a valid runtime.

Reply:
This is where I got it from: https://docs.flatpak.org/en/latest/available-runtimes.html?highlight=runtimes#freedesktop

Are the docs incorrect?

Reply:
```suggestion
runtime: org.freedesktop.Platform
```

It says **extension**

Reply:
@hfiguiere You are not exactly being helpful here. It also says *"Available Freedesktop runtimes:"* right before that list. It is not explained what an extension means, and I assumed we need that if we need GL support (which we do). Is that not the case?